tee-worker/local-setup/launch.py

```diff
@@ -59,11 +59,12 @@ def setup_worker(work_dir: str, source_dir: str, std_err: Union[None, int, IO],
 
 
 def run_worker(config, i: int, log_config_path):
-    log = open(f"{log_dir}/worker{i}.log", "w+")
+    id = config.get('id', i)
```

Comment:
Why not just use `i` which is being passed as parameter?

Reply:
`i` is treated as default but you can explicitly specify another value, even if you don't run the script multiple times at same time so it will be easier to find a logs for a worker if you run multiple at once.